### ISSUE TITLE
[@nativescript/camera] NativeClass is not defined

### ISSUE THREAD

**Issue by brysem** [NativeScript/plugins#72]
After updating NativeScript the camera package is crashing the app. This error is common when ts-patch is not being ran properly for a package build replacing the NativeClass decorator. The latest version 5.0.4 of @nativescript/camera seems to have this issue.

## Versions
```
✔ Component nativescript has 7.1.2 version and is up to date.
✔ Component @nativescript/core has 7.1.4 version and is up to date.
✔ Component @nativescript/ios has 7.1.1 version and is up to date.
✔ Component @nativescript/android has 7.0.1 version and is up to date.

@nativescript/camera@5.0.4
```

## Crash
```
***** Fatal JavaScript exception - application has been terminated. *****
NativeScript encountered a fatal error: Uncaught ReferenceError: NativeClass is not defined
at
../node_modules/@nativescript/camera/index.js(file: app/packages/camera/index.ios.ts:4:6)
at __webpack_require__(file: app/webpack/bootstrap:816:0)
at fn(file: app/webpack/bootstrap:120:0)
at ../node_modules/@acme/nativescript-core/image.js(file:///app/vendor.js:765:78)
at __webpack_require__(file: app/webpack/bootstrap:816:0)
at fn(file: app/webpack/bootstrap:120:0)
at ../node_modules/@acme/nativescript-core/index.js(file: app/packages/nativescript-core/index.ios.ts:1:0)
at __webpack_require__(file: app/webpack/bootstrap:816:0)
at fn(file: app/webpack/bootstrap:120:0)
at ../node_modules/ts-loader/index.js?!../node_modules/vue-loader/lib/index.js?!./components/App.vue?vue&type=script&lang=ts&(file:///app/bundle.js:1555:83)
at __webpack_require__(file: app/webpack/bootstrap:816:0)
at fn(file: app/webpack/bootstrap:120:0)
at ./components/App.vue?vue&type=script&lang=ts&(file: app/components/App.vue?2c2a:1:0)
at __webpack_require__(file: app/webpack/bootstrap:816:0)
at<…>
(CoreFoundation) *** Terminating app due to uncaught exception 'NativeScript encountered a fatal error: Uncaught ReferenceError: NativeClass is not defined
at
../node_modules/@nativescript/camera/index.js(file: app/packages/camera/index.ios.ts:4:6)
at __webpack_require__(file: app/webpack/bootstrap:816:0)
at fn(file: app/webpack/bootstrap:120:0)
at ../node_modules/@acme/nativescript-core/image.js(file:///app/vendor.js:765:78)
at __webpack_require__(file: app/webpack/bootstrap:816:0)
at fn(file: app/webpack/bootstrap:120:0)
at ../node_modules/@acme/nativescript-core/index.js(file: app/packages/nativescript-core/index.ios.ts:1:0)
at __webpack_require__(file: app/webpack/bootstrap:816:0)
at fn(file: app/webpack/bootstrap:120:0)
at ../node_modules/ts-loader/index.js?!../node_modules/vue-loader/lib/index.js?!./components/App.vue?vue&type=script&lang=ts&(file:///app/bundle.js:1555:83)
at __webpack_require__(file: app/webpack/bootstrap:816:0)
at fn(file: app/webpack/bootstrap:120:0)
at ./components/App.vue?vue&type=script&lang=ts&(file: app/components/App.vue?2c2a:1:0)
at __webpa<…>
```

## Temporary Solution
Downgrading to version 5.0.3 from 5.0.4. However, doing so, we don't get the Android 11 fixes.

**Comment by kfw9257** [comment]
I'm experiencing the same issue with version 5.0.5

**Comment by crowers** [comment]
Ditto

**Comment by NathanWalker** [comment]
This is resolved in `5.0.6`
